packages/ckeditor5-table/src/index.ts

```diff
@@ -29,3 +29,36 @@ export { default as TableSelection } from './tableselection';
 export { default as TableUtils } from './tableutils';
 export { default as TableColumnResize } from './tablecolumnresize';
 export { default as TableColumnResizeEditing } from './tablecolumnresize/tablecolumnresizeediting';
+
+export type { TableConfig } from './tableconfig';
+export type { default as InsertColumnCommand } from './commands/insertcolumncommand';
```

Comment:
Why only export types? AFAIK we usually export entire commands. They are a public API of a feature (both, as registered in commands registry and classes that can be reused).

Reply:
F2F discussion: It turned out that we didn't expose any commands so far in index.js files. I thought we did expose some already. Hence, let's continue not exposing anything for now (except types if needed).